web/src/routes/__tests__/PoisPage.spec.tsx

```diff
@@ -33,11 +35,10 @@ describe('PoisPage', () => {
   const poi0 = pois[0]!
   const poi1 = pois[1]!
 
-  const routePattern = `/:cityCode/:languageCode/${RoutePatterns[POIS_ROUTE]}`
+  const pathname = cityContentPath({ route: POIS_ROUTE, cityCode: city.code, languageCode: language.code })
 
-  const renderPois = ({ id }: { id?: string } = {}) => {
-    const pathname = cityContentPath({ route: POIS_ROUTE, cityCode: city.code, languageCode: language.code, path: id })
-    return renderRoute(
+  const renderPois = (
+    <ThemeProvider theme={buildConfig().lightTheme}>
```

Comment:
I think I fixed this already but please use the utility render method and just pass `{ wrapWithTheme: true }` as second argument

Reply:
this gonna be also fixed in your pr

Reply:
yeah thats what I wrote, just wanted to let you know :D